modules/op-starting-pipelines.adoc

```diff
@@ -19,7 +19,24 @@ For Pipelines created using the *From Git* option, the *Start Pipeline* dialog b
 ====
 +
 .. If you have resources in your namespace, the *Git Resources* and the *Image Resources* fields are prepopulated with those resources. If required, use the drop-downs to select or create the required resources and customize the Pipeline Run instance.
-.. If required, modify the *Advanced Options* and add credentials to authenticate the specified Git server or Docker registry.
+. Optional: Modify the *Advanced Options* to add credentials to authenticate the specified Git server or Docker registry.
```

Comment:
@nkakkar81 the structure looks great thank you!
the specified private Git server?